File: CS/ToolTipControllerExample/DetailControl.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Linq;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace ToolTipControllerExample {
    public partial class DetailControl : DevExpress.XtraEditors.XtraUserControl {
        public DetailControl() {
            InitializeComponent();
        }
        public void UpdateContent(Record record) {
            imageEdit.SvgImage = record.Image;
            stateToggleSwitch.EditValue = record.State;
            infoMemoEdit.EditValue = record.Info;
        }
    }
}

File: VB/ToolTipControllerExample/DetailControl.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Linq
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace ToolTipControllerExample
	Partial Public Class DetailControl
		Inherits DevExpress.XtraEditors.XtraUserControl

		Public Sub New()
			InitializeComponent()
		End Sub
		Public Sub UpdateContent(ByVal record As Record)
			imageEdit.SvgImage = record.Image
			stateToggleSwitch.EditValue = record.State
			infoMemoEdit.EditValue = record.Info
		End Sub
	End Class
End Namespace
